### ISSUE TITLE
[0.4.18 All] Multiple Monitor Support

### ISSUE THREAD

**Issue by teriflix** [teriflix/scrite#180]
Multiple writers are suggesting that we support multiple monitors. Perhaps screenplay window on one monitor, structure on another, notebook on yet another.

**Comment by suryavasishta** [comment]
This leads to the concept of 'workspaces' that Adobe employs. I can have multiple such window configurations stored under different workspace names. Selecting any workspace from a dropdown will autoload the window configuration. This I have predominantly used in Premiere and Aftereffects and it makes the whole process immensely convenient.

**Comment by pnudupa** [comment]
@suryavasishta - yes, I can see how this will be super useful.

**Comment by pnudupa** [comment]
@suryavasishta: Recent forum thread created by a user: https://www.scrite.io/index.php/forum/topic/dual-monitor-support/

**Comment by suryavasishta** [comment]
@pnudupa Yes I am aware of the forum topic. I thought this thread came here because of that !!
